### ISSUE TITLE
Dev Backend button doesn't show errors

### ISSUE THREAD

**Issue by niftynei** [zulip/zulip-android#80]
I have two suggestions for improvement with the Dev Backend button.

1) There's no spinner when you click on the "Dev Backend testing server" button to let you know that it's talking to the server. 
2) If there's any errors that occur after clicking "Dev Backend testing server", they aren't shown

cc @kunall17  :)

**Comment by niftynei** [comment]
Another thing: when I logged in as `iago` it spun forever on the client.  The server had this error message:

```
{"msg":"You are not authorized to get events from this queue", "result":"error"}
```

**Comment by kunall17** [comment]
@niftynei `{"msg":"You are not authorized to get events from this queue", "result":"error"}` This is weird I never encountered this message? Were you logged in before and then logout and tried this?

**Comment by niftynei** [comment]
@kunall17 Yep!  It's possible something on my backend is configured incorrectly.

**Comment by kunall17** [comment]
_2) If there's any errors that occur after clicking "Dev Backend testing server", they aren't shown_
fixed by #69

**Comment by niftynei** [comment]
@kunall17 I haven't checked, but I verified that errors are shown now, so it's not a high priority.

what are your thoughts on adding a spinner while the dev backend call is run?
